palace/drivers/eigensolver.cpp

```diff
@@ -300,6 +301,11 @@ EigenSolver::Solve(const std::vector<std::unique_ptr<Mesh>> &mesh) const
     // Compute B = -1/(iω) ∇ x E on the true dofs, and set the internal GridFunctions in
     // PostOperator for all postprocessing operations.
     eigen->GetEigenvector(i, E);
+    if (i < iodata.solver.eigenmode.n)
+    {
+      // Only consider the desired number of modes for the error indicator.
+      estimator.AddErrorIndicator(E, indicator);
```

Comment:
Note: I realized this is incorrect, since below the indicator is postprocessed for `i == 0` which means the value written isn't actually considering all modes. I should move it back outside the loop.

Reply:
Fixed in [95dbe4d](https://github.com/awslabs/palace/pull/168/commits/95dbe4d5c221dd33e0c8ff4df0ba5e770ffda0cf)